Common/Utilities.cs

```diff
@@ -2934,24 +2927,18 @@ public static void UploadFilesToContainer(string connectionString, string contai
                     throw;
                 }
 
-                // Create a blob client for interacting with the blob service.
-                var blobClient = storageAccount.CreateCloudBlobClient();
-
                 // Create a container for organizing blobs within the storage account.
                 Utilities.Log("1. Creating Container");
-                var container = blobClient.GetContainerReference(containerName);
+                var container = blobServiceClient.GetBlobContainerClient(containerName);
                 container.CreateIfNotExistsAsync().GetAwaiter().GetResult();
 
-                var containerPermissions = new BlobContainerPermissions();
-                // Include public access in the permissions object
-                containerPermissions.PublicAccess = BlobContainerPublicAccessType.Container;
                 // Set the permissions on the container
-                container.SetPermissionsAsync(containerPermissions).GetAwaiter().GetResult();
+                container.SetAccessPolicyAsync(PublicAccessType.BlobContainer).GetAwaiter().GetResult();

```

Comment:
GetAwaiter().GetResult() added here just trying to obey the consistency rule. Since many other places in this PR are using asynchronous method with GetAwaiter().GetResult().

What do you think about keeping GetAwaiter().GetResult().

Reply:
That code can deadlock, though not the worst way to do async over sync. Either use the sync methods or, if these are all track 1, use async throughout. Our samples should show best practices, or at the very least not violate common guidelines like calling async code this way. If the rest of the code needs to be updated, it's probably worth it to do so.

Reply:
The purpose here is doing sync while some methods only provide async api.
So what i am tring to do is that if there were sync api provided, sync api would be used. Otherwise, using async api with `ConfigureAwait(false)` added.
Details please refer to the new iteration. What do you think about this kind approach?

Reply:
After think it over, we decided to use `await` to call async code.
Could you help figure out if this is better?

Reply:
Yes, you should use `await` for async code.